src/canvas-extractor/tests/graphql/unit/grades/test_extractor_grades.py

```diff
@@ -38,3 +40,59 @@ def test_gql_grades_not_empty(
     )
 
     assert udm_grades is not None
+
+
+def test_gql_grades_duplicates(
+    mock_gql,
+    test_db_fixture
+):
+    sections = _get_sections(mock_gql, test_db_fixture)
```

Comment:
SHOULD: The population of the variables can probably be extracted as it is being used in the previous test as well.